### ISSUE TITLE
Download to sd-card does not start

### ISSUE THREAD

**Issue by C4R2T3N** [theScrabi/NewPipe#22]
If i change the download path to a folder on the sd-card, for example to:
"storage/sdcard1/Videos" (typing error excluded, i copied path)
The download does not start.

**Comment by theScrabi** [comment]
Jep. Download feature is very experimental, and working pretty bad right now. I really need to rework that.

**Comment by TheLastProject** [comment]
For what it's worth, this is what logcat says:
```
D/class org.schabi.newpipe.ActionBarHandler( 4609): bla
```

**Comment by theScrabi** [comment]
Ah yes. That tells me I didn't remove all debug Log functions. I guess that would help much :/

**Comment by timofonic** [comment]
Any news?

**Comment by theScrabi** [comment]
No. Not until we rewrite the download part.

**Comment by satiricon** [comment]
I tested this and it looks like it's not a problem anymore after GigaGet's download manager code was merged into NewPipe. I tested downloading to a sdcard and using the /storage/sdcard... paths and it worked just fine.
